pkgdiff

```diff
@@ -3,8 +3,14 @@
 
 set -e -x
 
+ignore_git_dir=${1}
+if [ "${ignore_git_dir}" = '--ignore-git-dir' ]; then
+	exclude_pattern=".git"
```

Comment:
Doesn't this match e.g. `.gitignore` too? I've never used `--exclude` tbh.

Also, what do you think about just doing this unconditionally? I don't think there's much reason to look inside git repo objects ;-).

Reply:
> Doesn't this match e.g. .gitignore too? I've never used --exclude tbh.

Looks like it does not. Made a quick repo with two commits after init one. One added .gitignore, the other added a line in gitignore:

```
diff --git a/.gitignore b/.gitignore
index 9daeafb..b02def2 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1 +1,2 @@
 test
+test2
```
Made a test ebuild with:

* Version 1 - init commit
* Version 2 - commit that added gitignore
* Version 3 - commit that added a line to gitignore

The patched  pkgdiff detects the gitignore change:
```shell
pkgdiff --ignore-git-dir test-diff-2.ebuild test-diff-3.ebuild
...
diff -dupr '--exclude=.git' /var/tmp/portage/app-portage/test-diff-2/work/test-diff-2/.gitignore /var/tmp/portage/app-portage/test-diff-3/work/test-diff-3/.gitignore
--- /var/tmp/portage/app-portage/test-diff-2/work/test-diff-2/.gitignore        2021-02-28 19:59:10.642345945 -0800
+++ /var/tmp/portage/app-portage/test-diff-3/work/test-diff-3/.gitignore        2021-02-28 19:59:12.241377640 -0800
@@ -1 +1,2 @@
 test
+test2
```

> Also, what do you think about just doing this unconditionally? I don't think there's much reason to look inside git repo objects ;-).

I personally am using `alias pkgdiff='pkgdiff --ignore-git-diff'` for my ebuild user. I can't think of a reason to look into git directory from this tool either. If the user wants to see the commit difference, they are better off using standard `git` tools. 

I was hesitant to change the tool's default behavior without a flag present. If you think the exclude should be there by default - just let me know, I'll update the PR.